operator/.golangci.yml

```
@@ -0,0 +1,59 @@
+    - nakedret
+    - prealloc
+    - rowserrcheck
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+    - whitespace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)